lms/envs/common.py

```diff
@@ -2295,3 +2295,6 @@
 PROFILE_IMAGE_SECRET_KEY = 'placeholder secret key'
 PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
 PROFILE_IMAGE_MIN_BYTES = 100
+
+# This is to check the domain in case of preview.
+PREVIEW_DOMAIN = 'preview'
```

Comment:
@muzaffaryousaf I'm looking at this for an Open edX question, and I'm now wondering why a new configuration setting had to be introduced. It means that to change the domain the developer has to change both ``PREVIEW_DOMAIN`` and ``PREVIEW_LMS_BASE``. Would it be possible to change this to remove the new configuration and just use ``PREVIEW_LMS_BASE`` instead?